plugin/core/views.py

```diff
@@ -939,66 +939,51 @@ def format_diagnostic_for_html(
     return "".join(formatted)
 
 
-def _is_completion_item_deprecated(item: CompletionItem) -> bool:
-    if item.get("deprecated", False):
-        return True
-    tags = item.get("tags")
-    if isinstance(tags, list):
-        return CompletionItemTag.Deprecated in tags
-    return False
-
-
-def _wrap_in_tags(tag: str, item: str) -> str:
-    return "<{0}>{1}</{0}>".format(tag, html.escape(item))
-
-
 def format_completion(
     item: CompletionItem, index: int, can_resolve_completion_items: bool, session_name: str
 ) -> sublime.CompletionItem:
     # This is a hot function. Don't do heavy computations or IO in this function.
-    item_kind = item.get("kind")
-    kind = COMPLETION_KINDS.get(item_kind, KIND_UNSPECIFIED) if item_kind else KIND_UNSPECIFIED
-
-    if _is_completion_item_deprecated(item):
-        kind = (kind[0], '!', "⚠ {} - Deprecated".format(kind[2]))
-
-    lsp_label = item["label"]
-    lsp_label_details = item.get("labelDetails")
-    lsp_filter_text = item.get("filterText")
-    st_annotation = (item.get("detail") or "").replace('\n', ' ')
-
-    st_details = ""
-    if can_resolve_completion_items or item.get("documentation"):
-        st_details += make_command_link("lsp_resolve_docs", "More", {"index": index, "session_name": session_name})
-    if lsp_label_details:
-        if st_details:
-            st_details += " | "
-        lsp_label_detail = lsp_label_details.get("detail")
-        lsp_label_description = lsp_label_details.get("description")
-        st_details += "<p>"
-        # `label` should be rendered most prominent.
-        st_details += _wrap_in_tags("b", lsp_label)
-        if isinstance(lsp_label_detail, str):
-            # `detail` should be rendered less prominent than `label`.
-            # The string is appended directly after `label`, with no additional white space applied.
-            st_details += html.escape(lsp_label_detail)
-        if isinstance(lsp_label_description, str):
-            # `description` should be rendered less prominent than `detail`.
-            # Additional separation is added.
-            st_details += " - {}".format(_wrap_in_tags("i", lsp_label_description))
-        st_details += "</p>"
-    elif lsp_filter_text and lsp_filter_text != lsp_label:
-        if st_details:
-            st_details += " | "
-        st_details += _wrap_in_tags("p", lsp_label)
+
+    lsp_label = item['label']
+    lsp_label_details = item.get('labelDetails') or {}
+    lsp_label_detail = lsp_label_details.get('detail') or ""
+    lsp_label_description = lsp_label_details.get('description') or ""
+    lsp_filter_text = item.get('filterText') or ""
+    lsp_detail = (item.get('detail') or "").replace("\n", " ")
+
+    kind = COMPLETION_KINDS.get(item.get('kind', 0), KIND_UNSPECIFIED)
```

Comment:
Just a style but the `0` fallback seems unnecessary here. The default `None` should work the same and IMO be more self-explanatory?

Reply:
The reason for it is because Pyright doesn't like it...

![pyright](https://user-images.githubusercontent.com/6579999/182045427-7b6ada0a-990d-4aea-938b-b4dc349776a7.png)

Reply:
Oh, OK.
Though maybe -1 would be more obvious then.